src/main/java/org/jabref/gui/maintable/columns/FileColumn.java

```diff
@@ -138,10 +138,20 @@ private void setCommonSettings() {
     }
 
     private String createFileTooltip(List<LinkedFile> linkedFiles) {
-        if (!linkedFiles.isEmpty()) {
-            return Localization.lang("Open file %0", linkedFiles.getFirst().getLink());
+        if (linkedFiles.isEmpty()) {
+            return null;
+        }
+
+        StringBuilder sb = new StringBuilder();
+
+        for (LinkedFile linkedFile : linkedFiles) {
+            sb.append("\n").append(linkedFile.getLink());
+        }
```

Comment:
Please use StringJoiner instead of StringBuilder (more modern - automatically adds `\n` if used correctly)